frontend/src/store/reducers/articleReducers.js

```diff
@@ -0,0 +1,56 @@
+import {
+	ADD_ARTICLE,
+	DELETE_ARTICLE,
+	FETCH_ARTICLE_DATA,
+	ARTICLE_DATA_FETCHED,
+	ERROR
+} from '../actions';
+
+const initialState = {
+	articles: [],
+	fetching: false,
+	success: false,
+	error: null
```

Comment:
Using `null` here gets a bit dicey when using prop-types, because prop-types doesn't have a null or undefined property check. Consider an empty string perhaps?